assets/js/modules/analytics-4/components/widgets/TopRecentTrendingPages.stories.js

```diff
@@ -26,12 +26,67 @@ import { KM_ANALYTICS_TOP_RECENT_TRENDING_PAGES } from '../../../../googlesiteki
 import { provideModules } from '../../../../../../tests/js/utils';
 import { withWidgetComponentProps } from '../../../../googlesitekit/widgets/util';
 import WithRegistrySetup from '../../../../../../tests/js/WithRegistrySetup';
-import TopRecentTrendingPagesWidget from './TopRecentTrendingPagesWidget';
+import TopRecentTrendingPages from './TopRecentTrendingPages';
 import { provideCustomDimensionError } from '../../utils/custom-dimensions';
+import {
+	STRATEGY_ZIP,
+	getAnalytics4MockResponse,
+	provideAnalytics4MockReport,
+} from '../../utils/data-mock';
+import { getDateString, getPreviousDate } from '../../../../util';
+import { replaceValuesInAnalytics4ReportWithZeroData } from '../../../../../../.storybook/utils/zeroReports';
 
 const WidgetWithComponentProps = withWidgetComponentProps(
 	KM_ANALYTICS_TOP_RECENT_TRENDING_PAGES
-)( TopRecentTrendingPagesWidget );
+)( TopRecentTrendingPages );
+
+const today = new Date();
+const todayDateString = getDateString( today );
+
+const yesterday = getPreviousDate( todayDateString, 1 );
+const twoDaysAgo = getPreviousDate( todayDateString, 2 );
+const threeDaysAgo = getPreviousDate( todayDateString, 3 );
+
+const reportOptions = {
+	startDate: threeDaysAgo,
+	endDate: yesterday,
+	dimensions: [ 'pagePath' ],
+	dimensionFilters: {
+		'customEvent:googlesitekit_post_date': {
+			filterType: 'inListFilter',
+			value: [
+				yesterday.replace( /-/g, '' ),
+				twoDaysAgo.replace( /-/g, '' ),
+				threeDaysAgo.replace( /-/g, '' ),
+			],
+		},
+	},
+	metrics: [ { name: 'screenPageViews' } ],
+	orderby: [
+		{
+			metric: {
+				metricName: 'screenPageViews',
+			},
+			desc: true,
+		},
+	],
+	limit: 3,
+};
```

Comment:
These should be exported from the component's file so the story can reuse them, rather than specifying them again here.

Reply:
@tofumatt Great point, thanks. I extracted it into a new dir, so other custom dimensions dependant widgets can also use it.